Resources/Prototypes/Entities/Structures/Doors/Airlocks/base_structureairlocks.yml

```diff
@@ -19,6 +19,8 @@
     - state: bolted_unlit
       shader: unshaded
       map: ["enum.DoorVisualLayers.BaseBolted"]
+    - state: emergency_unlit
+      map: ["enum.DoorVisualLayers.BaseEAccess"]
```

Comment:
The one in shuttle.yml is unshaded. Is there some reason this one isn't or was it just a mistake?

Reply:
I left it shaded as a mistake.